packages/ses/test/test-tolerate-empty-prototype.js

```diff
@@ -0,0 +1,25 @@
+import test from 'ava';
+import '../index.js';
+
+// See https://github.com/zloirock/core-js/issues/1092
+const originalPush = Array.prototype.push;
+// eslint-disable-next-line no-extend-native
+Array.prototype.push = function push(...args) {
+  return Reflect.apply(originalPush, this, args);
+};
+
+lockdown();
+
+test('tolerate empty prototype', t => {
+  t.assert('prototype' in Array.prototype.push);
+  t.is(Array.prototype.push.prototype, undefined);
```

Comment:
`Array.prototype.push.prototype` here is not set no `undefined`, so it's an object, no?

Reply:
It has been set to undefined here

https://github.com/endojs/endo/pull/1221/files/a1a133061c16cc47a1d813d5de87ed64472cd89b#diff-5e6b7c72e8b89f413a07218298c5926213a591b9e5164e52bf4e68b37bdbe4e2R289

Reply:
Ah, clear.